man/bootctl.xml

```diff
@@ -92,7 +92,10 @@
         url="https://systemd.io/BOOT_LOADER_SPECIFICATION">Boot Loader Specification</ulink>, as well as any
         other entries discovered or automatically generated by a boot loader implementing the <ulink
         url="https://systemd.io/BOOT_LOADER_INTERFACE">Boot Loader
-        Interface</ulink>.</para></listitem>
+        Interface</ulink>.</para>
+
+        <para>When combined with <option>--json=</option> outputs the boot loader entries in a series of
+        simple JSON objects.</para></listitem>
```

Comment:
Maybe `JSON output may be requested with <option>--json=</option>.

("object" is not something that users care about.)